src/extensions/links.ts

```diff
@@ -349,3 +349,13 @@ function prometheusSpecialRegexEscape<T>(value: T) {
     .replace(/\\/g, '\\\\\\\\') // escape backslashes
     .replace(/[$^*{}\[\]+?.()|]/g, '\\\\$&'); // escape regex metacharacters
 }
+
+// Need to export this function from scenes because importing scenesUtils is increasing the bundle entry point size by 522.51kB
+export function escapeUrlPipeDelimiters(value: string | undefined): string {
```

Comment:
https://github.com/grafana/scenes/pull/1230 to export this function

Reply:
Thank you for that issue, and the helpful code comment!